pkg/skaffold/test/structure/structure.go

```diff
@@ -86,7 +89,10 @@ func (cst *Runner) runStructureTests(ctx context.Context, out io.Writer, imageTa
 	for _, f := range files {
 		args = append(args, "--config", f)
 	}
-
+	for _, customArg := range cst.structureTestArgs {
+		splittedCustomArg := strings.Fields(customArg)
```

Comment:
We require the user to be responsible for splitting arguments properly.  Otherwise we need to deal with quoting issues, and YAML already makes that hard enough.  Docker uses this approach too.